src/swap/components/SwapProvider.tsx

```diff
@@ -139,13 +133,20 @@ export function SwapProvider({
 
         destination.setAmount(formattedAmount);
       } catch (err) {
-        handleError({ quoteError: err as SwapError });
+        setLifeCycleStatus({
+          statusName: 'error',
+          statusData: {
+            code: 'TmSPc01', // Transaction module SwapProvider component 01 error
```

Comment:
Hmm I don't love this error code

Reply:
This is going to be the new starnard for all error code, where the goal is to create location from where the error is happening.

See error code as longitude and latitude.

So in this case TmSPc01 means
Tm = Transaction module
SPc = Swap Provider component
01 = First error of the file

Reply:
I am going to rewrite all the error code to be more like this, and instead have the Message and Error value be more about description on what's up.

Reply:
Have an error code as location, help debugging, and support developers when something is broken, as the first thing we will check, it's where the Error Code is originated.

Reply:
Interesting, ok got it! Once all error codes match this format i'm sure I'll get used to it quickly

Reply:
cc @abcrane123 @cpcramer @0xAlec